pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -729,6 +731,7 @@ func (c *Cluster) getOSDPropsForPVC(pvcName string) (osdProperties, error) {
 				pvcSize:             volumeSource.Size,
 				schedulerName:       volumeSource.SchedulerName,
 				encrypted:           volumeSource.Encrypted,
+				name:                volumeSource.Name,
```

Comment:
Isn't this just the pvc.ClaimName? Why do we need a new entry?

Reply:
`pvc.ClaimName` =`set1-data-2-wr6h7` and `name`=`set1` . So,we can say `name` is subset  of `pvc.ClaimName` but it is not same and we want labels according to the `storageClassDeviceSet` i.e `name` in above case.